src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -791,11 +791,12 @@ private void TextBoxBase_OnTextChanged(object sender, TextChangedEventArgs e)
         /// </summary>
```

Comment:
`DynamoModel.PathManager.SamplesDirectory` now points to the per-instance sample folder.